### PR TITLE
move deps in root cargo.toml

### LINKED ISSUE #134: hoist the Cargo.toml values into the workspace Cargo.toml
ATM, the dependencies are duplicated in each workspace crate. Instead those should be listed in the root Cargo.toml. The same applies to licenses and other stuff.

### PR DESCRIPTION
**Pull request by gabriele-0201**
closes #134

### CODE REVIEW

**Comment by gabriele-0201** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #195** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/195?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/thrumdev/blobs/195?utm_source=stack-comment).

**Comment by rphmeier** [comment]
@pepyakin can you review this one? The changes look like they'd work, to me, but does it accomplish what you wanted it to?

File: Cargo.toml [hunk 1]
```diff
@@ -23,3 +23,155 @@ members = [
     "sugondat/shim/common/sovereign",
     "sugondat/subxt-autogen"
 ]
+
+[workspace.package]
+authors = ["Robert Habermeier - @rphmeier" , "Sergei Shulepov - @pepyakin", "Gabriele Miotti - @gabriele-0201"]
```
Comment (pepyakin) [inline Cargo.toml:28]: A couple of things. I think this field is conventionally has the same format as name and email.

Quote from Cargo docs:

```
[package]
# ...
authors = ["Graydon Hoare", "Fnu Lnu <no-reply@rust-lang.org>"]
```

However, I would rather not put any names here. It's just not scalable, may introduce some unnecessary feelings, and most importantly serves no purpose! 

I see the following ways:

1. remove it altogether. I don't think it's required.
2. use unpersonified email, similar to what we do in Parity. smth like `parity developers <admin@parity.io>`.

**Review by pepyakin — CHANGES_REQUESTED** [review]
Yes, this is exactly what I meant.

**Review by pepyakin — APPROVED** [review]

**Pull request merged**